Content.Shared/Interaction/SharedInteractionSystem.cs

```diff
@@ -1372,8 +1371,6 @@ public void DoContactInteraction(EntityUid uidA, EntityUid? uidB, HandledEntityE
             if (uidB == null || args?.Handled == false)
                 return;
 
-            DebugTools.AssertNotEqual(uidA, uidB.Value);
-
```

Comment:
And this one I guess should just be changed to a return statement. But ill just open a separate PR to do this & fix plant holders